CHANGELOG.md

```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [PEP 440](https://www.python.org/dev/peps/pep-0440/)
 and uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+### New/Updated Features
++ Support temporal interpolation when user requested time is not exactly on a weather model time
```

Comment:
Are you allowing for an input option where this is done? aka it should be configurable whether or not to do temporal interpolation or jumping to nearest time.

I did not see in this commit below where the function is called within raider itself.

Reply:
It was previously kept separate via a new entry point. It's now embedded directly in RAiDER.py:calcDelay